cumulus_library/databases/athena.py

```diff
@@ -70,7 +70,7 @@ def execute_as_pandas(
     def parser(self) -> base.DatabaseParser:
         return AthenaParser()
 
-    def operational_errors(self) -> tuple[Exception]:
+    def operational_errors(self) -> tuple[type[Exception], ...]:
```

Comment:
Unrelated, but I noticed that this method had incorrect typing (I think my fault).